lib/src/upgrade_messages.dart

```diff
@@ -161,7 +161,7 @@ class UpgraderMessages {
       case 'en':
       default:
         message =
-            'A new version of {{appName}} is available! Version {{currentAppStoreVersion}} is now available-you have {{currentInstalledVersion}}.';
+            'A new version of {{appName}} is available! Version {{currentAppStoreVersion}} is now available - you have {{currentInstalledVersion}}.';
```

Comment:
This change is not related to Android and the Play Store.

Reply:
I will fix the change for this also